README.md

```diff
@@ -27,26 +49,134 @@ int main( )
+- `Base64BinaryAppendedWriter`
+- `RawBinaryAppendedWriter`
+- `CompressedRawBinaryAppendedWriter` (if [zlib](https://zlib.net/) is available)
 
 ## Parallel example
 
-Dodo.
+The pvtu format is used in combination with the vtu format. The mesh needs to be partitioned before it is given to _vtu11_. Each part of the mesh is written to a vtu file, and the pvtu file contains the references to those files. Overlapping entities like ghost nodes or cells can be added too if needed for e.g. other cells.
+
+```cpp
+#include "vtu11.hpp"
+
+int main( )
+{
+    // Split the 3x2 cells from serial example in two 3x1 partitions
+
+    // Row 0 and 1
+    std::vector<double> points0
+    {
+        0.0, 0.0, 0.5,    0.0, 0.3, 0.5,    0.0, 0.7, 0.5,    0.0, 1.0, 0.5, // 0,  1,  2,  3
+        0.5, 0.0, 0.5,    0.5, 0.3, 0.5,    0.5, 0.7, 0.5,    0.5, 1.0, 0.5, // 4,  5,  6,  7
+    };
+
+    // Row 1 and 2
+    std::vector<double> points1
+    {                                                                        // Original indices:
+        0.5, 0.0, 0.5,    0.5, 0.3, 0.5,    0.5, 0.7, 0.5,    0.5, 1.0, 0.5, // 4,  5,  6,  7
+        1.0, 0.0, 0.5,    1.0, 0.3, 0.5,    1.0, 0.7, 0.5,    1.0, 1.0, 0.5  // 8,  9, 10, 11
+    };
+
+    // Original cells 0, 1 and 2
+    std::vector<vtu11::VtkIndexType> connectivity0
+    {
+        0,  4,  5,  1, // 0
+        1,  5,  6,  2, // 1
+        2,  6,  7,  3, // 2
+    };
+
+    // Original cells 3, 4, and 5 (now using local vertex indices)
+    std::vector<vtu11::VtkIndexType> connectivity1
+    {
+        0,  4,  5,  1, // 3
+        1,  5,  6,  2, // 4
+        2,  6,  7,  3, // 5
+    };
+
+    std::vector<vtu11::VtkIndexType> offsets0 { 4, 8, 12 };
+    std::vector<vtu11::VtkIndexType> offsets1 { 4, 8, 12 };
+
+    std::vector<vtu11::VtkCellType> types0 { 9, 9, 9 };
+    std::vector<vtu11::VtkCellType> types1 { 9, 9, 9 };
+
+    // Create one proxy for each partition
+    vtu11::Vtu11UnstructuredMesh meshPartition0 { points0, connectivity0, offsets0, types0 };
+    vtu11::Vtu11UnstructuredMesh meshPartition1 { points1, connectivity1, offsets1, types1 };
+
+    // Because vertices are duplicated also point data is duplicated
+    std::vector<double> pointData0 { 0.0, 1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0 };
+    std::vector<double> pointData1 { 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 11.0 };
+
+    std::vector<double> cellData0 { 3.2, 4.3, 5.4 };
+    std::vector<double> cellData1 { 6.5, 7.6, 8.7 };
+
+    // These hold for all partitions
+    std::vector<vtu11::DataSetInfo> dataSetInfo
+    {
+        { "Temperature", vtu11::DataSetType::PointData, 1 },
+        { "Conductivity", vtu11::DataSetType::CellData, 1 },
+    };
+
+    std::vector<vtu11::DataSetData> dataSetData0 { pointData0, cellData0 };
+    std::vector<vtu11::DataSetData> dataSetData1 { pointData1, cellData1 };
+
+    // Create writer
+    vtu11::RawBinaryAppendedWriter writer;
+
+    size_t numberOfFiles = 2;
+
+    std::string path = ".";
```

Comment:
I thought we wanted to drop this argument?
I suggest the following, both for the serial and the parallel version:
pass only one argument as name, can be as simple as `test` but can also be a path e.g. `path/to/awesome_results`
=> then internally we add the necessary file extensions (`vtu` & `pvtu`)

If you agree I can do the changes so that I also contributed a bit :)

Reply:
Haha :)

Yeah I guess we can. With filesystem you can even check whether something .has_extension( ). We could at least for the .vtu files allow both. For .pvtu idk if it makes sense, but I guess we could also allow it.

In my code I already split the path into 
```cpp
auto pathStr = path.parent_path( ).string( );
auto baseNameStr = path.stem( ).string( );
```